docs/application/native/guides/internationalization/i18n.md

```diff
@@ -773,6 +777,104 @@ The `current()` and `next()` functions only check the current index against the
 
 The UCharIter functions return code unit values in the range of 0 - 0xffff. Before any functions operating on strings are called, the string must be set with the `i18n_uchar_iter_set_string()`, `i18n_uchar_iter_set_UTF16BE()`, or `i18n_uchar_iter_set_UTF8()`function.
 
+
+<a name="plural_format"></a>
+## Formating plural messages with PluralFormat
+
+The PluralFormat API (in [mobile](../../api/mobile/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__BASE__UTILS__I18N__PLURAL__FORMAT__MODULE.html) applications) allows you to [format defined plural words](#plural_format_examples). Alongside with PluralRules module, you can define rules to which format would apply.
+
+Plural Format supports the creation of internationalized messages with plural inflection.
+It is based on plural selection, i.e. the caller specifies messages for each plural case
+that can appear in the user's language and the Plural Format selects the appropriate message
+based on the number.
+
+### The Problem of Plural Forms in Internationalized Messages:
```

Comment:
Please can you change as:

Problem of plural forms in internationalized messages